sdk/storage/Azure.Storage.DataMovement/src/TransferManager.cs

```diff
@@ -99,16 +100,20 @@ private void ConfigureProcessorCallbacks()
             _chunksProcessor.Process = Task.Run;
         }
 
-        private async Task ProcessJobAsync(TransferJobInternal job, CancellationToken _)
+        private async Task ProcessJobAsync(TransferJobInternal job, CancellationToken cancellationToken = default)
         {
+            cancellationToken = LinkCancellation(cancellationToken);
```

Comment:
Since this method and below are `private` is it safe to assume the cancellation token was already linked in whatever public method called these?